src/request_queue.js

```diff
@@ -26,6 +26,15 @@ export const API_PROCESSED_REQUESTS_DELAY_MILLIS = 10 * 1000;
 // How many times we try to get queue head with queueModifiedAt older than API_PROCESSED_REQUESTS_DELAY_MILLIS.
 export const MAX_QUERIES_FOR_CONSISTENCY = 6;
 
+// This number must be large enough so that processing of all the requests cannot be done in
+// time lower than maximum latency of DynamoDB, but not low enough to not cost too much memory.
```

Comment:
`not low enough to not cost too much memory` means `not too large to cost too much memory`?

Reply:
yes :) fixing it